brayns/common/scene/Model.cpp

```diff
@@ -118,10 +118,61 @@ void Model::addModel(ModelPtr model, const Transformations& transform)
     _modelsDirty = true;
 }
 
+uint64_t Model::addSDFGeometry(const size_t materialId, const SDFGeometry& geom,
+                               const std::vector<size_t>& neighbourIndices)
+{
+    _sdfGeometryData.isDirty = true;
+    const uint32_t geomIdx = _sdfGeometryData.geometries.size();
+    _sdfGeometryData.geometryIndices[materialId].push_back(geomIdx);
+
+    assert(_sdfGeometryData.geometries.size() ==
+           _sdfGeometryData.neighbours.size());
+
+    _sdfGeometryData.neighbours.push_back(neighbourIndices);
+    _sdfGeometryData.geometries.push_back(geom);
```

Comment:
To limit indirection and make the code more readable, the above could moved to a function of sdfGeometryData and have here:
_sdfGeometryData.add(materialId, geom, neighbourIndices);

Reply:
I renamed the variable to _sdf. Do you think it is readable enough then?

Reply:
hmm sure, although my point was not exactly about the variable name. What I was trying to say is more that since those lines of code operate exclusively on the struct members, it could make sense to put them a member function of the struct which is called it from here... but that's fine as well.